assets/app/views/_pods.html

```diff
@@ -1,11 +1,14 @@
 <!--
-  Expects the following variables:
-  pods - a hash of pod objects by id
+Expects the following variables:
+pods - a hash of pod objects by id
 -->
 <div class="component-block pod-container">
-  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod">
+  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod" ng-init="showWarning = (pod | isTroubledPod)" ng-class="{'pod-warning': showWarning}">
     <div class="pod-text">
-      <strong>{{pod.status.phase}}</strong><div ng-if="pod.status.podIP">{{pod.status.podIP}}</div><div ng-if="!pod.status.podIP">allocating IP...</div>
+      <!-- Add a tooltip only if showing a pod warning -->
+      <strong ng-if="showWarning" data-toggle="tooltip" title="Check pod and container statuses for problems">{{pod.status.phase}}</strong>
```

Comment:
A generic tooltip warning of "check container statuses for problems" doesn't seem to add enough insight. I was thinking it would show specifics to the error, 50 characters or less. Something like... 
"Waiting (Error: image library/sinatra-app-example:latest not found)" or some message like the one shown here  https://cloud.githubusercontent.com/assets/1167259/7708677/4c38a760-fe25-11e4-97db-ad75aa1aa54c.png